api/swagger.yaml

```diff
@@ -189,6 +189,48 @@ definitions:
       - CICD
       - RUNTIME
 
+  CISDockerBenchmarkLevelCount:
+    type: object
+    properties:
+      level:
+        $ref: '#/definitions/CISDockerBenchmarkLevel'
+      count:
+        type: 'integer'
+        format: 'uint32'
+
+  CISDockerBenchmarkLevel:
+    type: string
+    enum: &CISDockerBenchmarkLevel
+      - INFO
+      - WARN
+      - FATAL
+
+  CISDockerBenchmarkCodeInfo:
+    type: 'object'
+    properties:
+      code:
+        type: 'string'
+      level:
+        type: 'number'
+        format: integer
+      assessments:
+        type: 'array'
+        items:
+          $ref: '#/definitions/CISDockerBenchmarkAssessment'
+
+  CISDockerBenchmarkAssessment:
+    type: 'object'
+    properties:
+      code:
```

Comment:
why code and level is duplicated from CISDockerBenchmarkCodeInfo?

Reply:
not sure, I just copied it from the runtime scan api https://github.com/cisco-open/kubei/blob/main/runtime_scan/api/swagger.yaml#L86